prime-dotnet-webapi/Services/interfaces/IEnrolleePaperSubmissionService.cs

```diff
@@ -23,7 +24,7 @@ public interface IEnrolleePaperSubmissionService
         Task UpdateAgreementAsync(int enrolleeId, PaperEnrolleeAgreementViewModel viewModel);
         Task SetProfileCompletedAsync(int enrolleeId);
         Task FinalizeSubmissionAsync(int enrolleeId);
-        Task AddEnrolleeAdjudicationDocumentsAsync(int enrolleeId, int adminId, IEnumerable<Guid> documents);
+        Task AddEnrolleeAdjudicationDocumentsAsync(int enrolleeId, int adminId, IEnumerable<PaperEnrolleeDocumentViewModel> payload);
```

Comment:
I would keep the name of the parameter "documents" rather than "payload". I assume it was copy-pasted from the controller.

Reply:
I will revert to documents, I just thought payload would be more future proof if we add more parameters to what's being sent.